pkg/controller/controller_suite_test.go

```diff
@@ -555,6 +555,25 @@ func waitForCacheSync(stop <-chan struct{}, controller *controller) {
 	)).To(BeTrue())
 }
 
+func EqualTaints(a, b []corev1.Taint) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for _, va := range a {
+		found := false
+		for j := range b {
+			if va == b[j] {
```

Comment:
Does this comparison work for `Taint.TimeAdded` too?

Reply:
👍 Maybe a `DeepEquals` is better here.

Reply:
For test cases, we are not setting TimeAdded, and hence it should be fine. 
Although it's a good point,  I am not sure - how would it really work overall with taint feature.